### PR TITLE
feat(cli): change virtualenv directory to .venv to comply with python recommendation

### LINKED ISSUE #9134: [cli] Create Python virtual environment in .venv directory
`cdk init --language python` creates virtual environment in `.env` directory. The ask is to create the virtual environment in `.venv` directory.

### Use Case

Python [virtual environment](https://docs.python.org/3/tutorial/venv.html) documentation recommends using `.venv` directory, to prevent clashing with `.env` environment variable definition files that some tooling supports. `.venv` is also the default in last PyCharm versions.

### Proposed Solution

Modify CDK CLI to create virtual environment in `.venv` directory

### Other

* [ ] :wave: I may be able to implement this feature request
* [ ] :warning: This feature might incur a breaking change

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by AnderEnder**
Replace virtualenv directory `.env` to `.venv` for the python templates

Closes #9134

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk/README.md [hunk 1]
```diff
@@ -45,9 +45,11 @@ $ # List the available template types & languages
 $ cdk init --list
 Available templates:
 * app: Template for a CDK Application
-   └─ cdk init app --language=[java|typescript]
+   └─ cdk init app --language=[csharp|fsharp|java|javascript|python|typescript]
```
Comment (shivlaks) [inline packages/aws-cdk/README.md:48]: thanks for updating this!

**Review by shivlaks — COMMENTED** [review]
@AnderEnder changes look good! have you run the `yarn integ-init` or `npm run integ-init` script to verify that it still succeeds? (it does not get executed as part of the PR build)

**Comment by AnderEnder** [comment]
> have you run the yarn integ-init or npm run integ-init script to verify that it still succeeds? (it does not get executed as part of the PR build)

I was not able to run it even on master.
```
$ yarn integ-cli
yarn run v1.22.10
$ npm run integ-cli-regression && npm run integ-cli-no-regression

> aws-cdk@0.0.0 integ-cli-regression /Users/ander/projects/aws-cdk/packages/aws-cdk
> npm run integ-cli-regression-latest-release && npm run integ-cli-regression-latest-code


> aws-cdk@0.0.0 integ-cli-regression-latest-release /Users/ander/projects/aws-cdk/packages/aws-cdk
> test/integ/run-against-dist test/integ/test-cli-regression-against-latest-release.sh

/Users/ander/projects/aws-cdk/packages/aws-cdk does not seem to be a built CDK distribution (change directory or use DIST_ROOT)
/Users/ander/projects/aws-cdk/packages/aws-cdk/test/integ/run-against-dist.bash: line 11: TRAPS[@]: unbound variable
```

**Comment by shivlaks** [comment]
> > have you run the yarn integ-init or npm run integ-init script to verify that it still succeeds? (it does not get executed as part of the PR build)
> 
> Unfortunately, I was not able to run it even on master.
> 
> ```
> $ yarn integ-cli
> yarn run v1.22.10
> $ npm run integ-cli-regression && npm run integ-cli-no-regression
> 
> > aws-cdk@0.0.0 integ-cli-regression /Users/ander/projects/aws-cdk/packages/aws-cdk
> > npm run integ-cli-regression-latest-release && npm run integ-cli-regression-latest-code
> 
> 
> > aws-cdk@0.0.0 integ-cli-regression-latest-release /Users/ander/projects/aws-cdk/packages/aws-cdk
> > test/integ/run-against-dist test/integ/test-cli-regression-against-latest-release.sh
> 
> /Users/ander/projects/aws-cdk/packages/aws-cdk does not seem to be a built CDK distribution (change directory or use DIST_ROOT)
> /Users/ander/projects/aws-cdk/packages/aws-cdk/test/integ/run-against-dist.bash: line 11: TRAPS[@]: unbound variable
> ```

@AnderEnder - i had just the `yarn integ-init` target in mind as it'll only test the init templates. does that also fail for you?

**Comment by AnderEnder** [comment]
> @AnderEnder - i had just the yarn integ-init target in mind as it'll only test the init templates. does that also fail for you?

Unfortunately the result is similar:
```
$ yarn integ-init
yarn run v1.22.10
$ test/integ/run-against-dist test/integ/init/test-all.sh
/Users/ander/projects/aws-cdk/packages/aws-cdk does not seem to be a built CDK distribution (change directory or use DIST_ROOT)
/Users/ander/projects/aws-cdk/packages/aws-cdk/test/integ/run-against-dist.bash: line 11: TRAPS[@]: unbound variable
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by shivlaks** [comment]
> > @AnderEnder - i had just the yarn integ-init target in mind as it'll only test the init templates. does that also fail for you?
> 
> @shivlaks, unfortunately, the result is similar:
> 
> ```
> $ yarn integ-init
> yarn run v1.22.10
> $ test/integ/run-against-dist test/integ/init/test-all.sh
> /Users/ander/projects/aws-cdk/packages/aws-cdk does not seem to be a built CDK distribution (change directory or use DIST_ROOT)
> /Users/ander/projects/aws-cdk/packages/aws-cdk/test/integ/run-against-dist.bash: line 11: TRAPS[@]: unbound variable
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> ```

@AnderEnder thanks for giving it a shot, that is unfortunate. i'll try to pull this change down and see if I can run it. but as a side note, we probably need to give a little more love in this area so contributors can run them without so much friction...

**Review by shivlaks — APPROVED** [review]

**Pull request merged**